orchestrationSpecs/packages/schemas/src/userSchemas.ts

```diff
@@ -99,12 +99,40 @@ export const KAFKA_SERVICES_CONFIG = z.object({
 });
 
 export const S3_REPO_CONFIG = z.object({
-    awsRegion: z.string().describe("The AWS region that the bucket reside in (us-east-2, etc)"),
+    awsRegion: z.string()
+        .meta({
+            title: "AWS Region",
+            description: "The AWS region that the bucket resides in (us-east-2, etc)",
+            placeholder: "us-east-2",
+            order: 1
```

Comment:
Can you name this something else.  Would you say that this is the level of importance?  Why would these 4 items have this order?